js/components/ComponentFactory.js

```diff
@@ -214,6 +215,7 @@ define(function (require) {
             case 'LINKBUTTON': require(['./interface/linkButton/LinkButton'],cb); break;
             case 'BUTTONBAR': require(['./interface/buttonBar/ButtonBar'], cb); break;
             case 'DRAWER': require(['./interface/drawer/TabbedDrawer'], cb); break;
+            case 'PLOT': require(['./interface/plot/Plot'], cb); break;
```

Comment:
Wasn't already there? 😅

Reply:
Wtf I was also working with plots with Matt and I included some stuff here. Let's  get rid of that